instrument/Facilities.xml

```diff
@@ -701,7 +701,9 @@
   <instrument name="D22">
     <technique>SANS</technique>
   </instrument>
-
+  <instrument name="D22B">
```

Comment:
Is there a way around without this? D22B is just the upgrade of D22, ideally it should not appear in the facilities file.

Reply:
We have 2 different geometries for the same name, and LoadEventNexus has to decide which one to load based on something.

We could perhaps set the IDF path in the event file directly, and modify LoadEventNexus so that it uses this path if it is provided, but that would mean that for all of ILL event nexuses, the IDF path could not be changed *ever* afterward, which to me looks like a very bad idea.

The only other alternative I see is putting an if clause exclusively for D22 in LoadEventNexus, and it also seems pretty bad.

Reply:
I think the best is to write D22B as instrument name when converting to event.nxs. 
Then we can remove it from the facilities.xml.
Finally, in the load event nexus it can check if the retrieved instrument name exists in the facilities it should do what it does now, otherwise, it can construct the expected IDF path, and run `LoadInstrument` by giving the `Filename` instead of `InstrumentName`. You can check how `LoadILLSANS::getInstrumentFilePath` does it, for example.